helper-cli/src/main/kotlin/commands/ListCopyrightsCommand.kt

```diff
@@ -20,11 +20,15 @@
 package org.ossreviewtoolkit.helper.commands
 
 import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.groups.mutuallyExclusiveOptions
+import com.github.ajalt.clikt.parameters.groups.single
 import com.github.ajalt.clikt.parameters.options.convert
 import com.github.ajalt.clikt.parameters.options.flag
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.required
 import com.github.ajalt.clikt.parameters.types.file
+import org.ossreviewtoolkit.helper.common.PackageConfigurationOption
```

Comment:
Add an empty line above this import.

Reply:
Done.

Reply:
Looks like you did not push the changes.

Reply:
oh - now I did.